### DISCUSSION TITLE
Grafica voti genitori /alunni

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#91] [Idee e proposte]
Buongiorno ,sarebbe possibile avere per i genitori e gli alunni la seguente grafica per quanto riguarda la visualizzazione dei voti?
Cercando in giro ho trovato questo modello utilizzato da AXIOS e mi sembra geniale, intuitivo e perfetto oltre che di facile comprensione .

https://user-images.githubusercontent.com/93823125/147840723-26f98080-a851-48b7-8acf-5ed95013d0f9.png

https://user-images.githubusercontent.com/93823125/147840850-ea56e2ec-cb36-4e89-9b4e-9781b5a030c7.png

**Comment by trinko** [comment]
E' senz'altro una visualizzazione interessante. Al momento però non ho tempo per questo tipo di modifiche.
Lo segno per sviluppi futuri.
